src/filters/utils.cjs

```diff
@@ -65,4 +65,29 @@ const onlyShow = (array, n) => {
   return array.slice(0, n);
 };
 
-module.exports = { typeCheck, styles, onlyShow };
+/* @docs
+label: oddNewsTags
+category: Filter
+note: |
+  Returns the matching mailchimp form tag
+  for the page where the signup form lives |
+
+params:
+  name:
+    type: string
+    note: name of page such as 'footer'
+*/
+const oddNewsTags = (name) => {
+  switch (name) {
+    case 'footer':
+      return '6264369';
+    case 'oddblog':
+      return '6265233';
```

Comment:
It looks like this is unused, correct?

Reply:
Yes for now, but for the Sondra's new PWA article, we will use the form with this tag

Reply:
Just for that post, or for all blog posts?

Reply:
Makes sense -- any particular reason to wait for #323 for that change then?

Reply:
I think that article is waiting for this PR to be merged so that we can drop the sign up form in but seems like we're good to merge here